sydent/sydent.py

```diff
@@ -306,9 +307,45 @@ def run_gc():
             gc.collect(i)
 
 
+def setup_logging(config: SydentConfig) -> None:
+    """
+    Setup logging using the options specified in the config
+
+    :param config: the configuration to use
+    """
+    log_path = config.general.log_path
+    log_level = config.general.log_level
+
+    log_format = "%(asctime)s - %(name)s - %(lineno)d - %(levelname)s" " - %(message)s"
+    formatter = logging.Formatter(log_format)
+
+    if log_path != "":
+        handler: logging.StreamHandler = logging.handlers.TimedRotatingFileHandler(
+            log_path, when="midnight", backupCount=365
+        )
+        handler.setFormatter(formatter)
+
+        def sighup(signum, stack):
```

Comment:
this isn't your fault, but this doesn't seem to be used anywhere 🤦

Reply:
Oh I removed that at some point in a seperate PR but might as well do it here instead!

Reply:
it might have been better to leave it, and wire it up correctly, but never mind :)